src/vision/pi_vision/pi_face_tracker/nodes/face_tracker.py

```diff
@@ -258,7 +263,10 @@ def __repr__(self):
         )
 
     def update_bounding_box(self):
-        self.bounding_size = self.pt2[1] - self.pt1[1]
+        #is dp 0.22 the correct size for width or height of face?
+        #now using width instead of height as initialized
```

Comment:
Hi Linas,
 I figured out the 0.22 was for height of face, but we needed width of face
due to the original formulas. The value we set in cmt was 0.17m, but going
by the links you sent it should be something like 0.15 or 0.16.
Yes, there is a definite problem that we do not usually have the correct
bounding box and bounding box will vary a bit time to time, but I am not
sure how that can be solved without improved detection algorithms.
I guess we will be shifting to cmt from pi vision, it seems to have much
better tracking performance, but will update comments soon.
Regards,
Mandeep

On Fri, Jul 8, 2016 at 8:08 AM, Linas Vepštas <notifications@github.com>
wrote:

> In src/vision/pi_vision/pi_face_tracker/nodes/face_tracker.py
> <https://github.com/hansonrobotics/HEAD/pull/9#discussion_r70008906>:
>
> > @@ -258,7 +263,10 @@ def __repr__(self):
> >          )
> >
> >      def update_bounding_box(self):
> > -        self.bounding_size = self.pt2[1] - self.pt1[1]
> > +        #is dp 0.22 the correct size for width or height of face?
> > +        #now using width instead of height as initialized
>
> Of course, it might be the case that CMT is not actually placing the
> bounding box on the actual head, chin to crown, but on something smaller or
> larger? If the CMT bounding-box size is different from the size of the
> human head, then there would need to be a correction for that.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hansonrobotics/HEAD/pull/9/files/1b5f9e0c4c0f4a459b088785bb58273ffbb10a42#r70008906>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHM_f65quoFSORAWi0N53YEJI_e4pisbks5qTZT1gaJpZM4ImJGP>
> .
>